pkg/eval/mods/path/path_test.go

```diff
@@ -59,9 +63,20 @@ func TestPath(t *testing.T) {
 		That(`path:is-dir a/b/s`).Puts(false),
 		That(`path:is-dir `+tmpdir).Puts(true),
 		That(`path:is-dir s1`).Puts(false),
+		That(`path:is-dir d1/d`).Puts(false),
+		// WTF: This fails on Windows 10.
+		That(`path:is-dir d1/d &follow-symlink`).Puts(true),
```

Comment:
Please leave "WTF" out of the code. If you can't figure out why this fails in Windows 10, leave this line commented out and add a TODO to fix this test on Windows.

Reply:
The whole point of a "WTF" comment is to remind me of something that needs to be resolved before anyone else sees the change. All of the tests pass on Windows. So I need to figure out if I botched a `git master rebase`, simply overlooked the need to delete that comment, or something else. Will push a corrected change soon.

Reply:
It looks like removing this "WTF" comment was an oversight. I fixed the reason for this test failure on Windows but didn't remove this comment. Lines 19 to 22 of the change to this file explains why symlinks have to be created after the referent on Windows10.